test/component-bind.spec.ts

```diff
@@ -297,9 +299,16 @@ describe('component', () => {
     describe('with ">>" prefix', () => {
 
       @component class ToRight extends Composite {
+
         // space after direction is not required
```

Comment:
No need to demonstrate this edge case, as this sounds like the expected behavior?

Reply:
??